docs/products/cassandra/howto/enable-cross-cluster-replication.rst

```diff
@@ -0,0 +1,199 @@
+   3. Select a cloud provider and a cloud region for the second service.
+   4. Select one service plan for your two new services.
+   5. Optionally, add an additional disk storage for your services.
+   6. Enter names for your services.
+
+.. topic:: Datacenters names
+
+   Notice that the service names you enter are automatically copied as the names for datacenters for the services. The datacenters names can be required when executing statements on your databases.
+
+4. Verify your choices in the **Services summary** card on the right and, if your configuration turns out as expected, select **+Create services** if the setting looks.
+
+.. topic:: Result
+   
+   You've created two new services that are connected for CCR purposes. You can preview CCR details for your services in the **Overview** tab under **Cross Cluster replication**.
+
+On an existing service
+''''''''''''''''''''''
+
+.. important::
+
+    * Your existing service has ``NetworkTopologyStrategy`` set as a replication strategy for its keyspace.
+    * Your existing service and the service you connect to (``service_to_join_with``) are hosted on different datacenters.
+
+1. Log in to the `Aiven console <https://console.aiven.io/>`_.
+2. From the **Current services** view, select an Aiven for Apache Cassandra service on which you'd like to enable CCR.
+3. In the **Overview** tab of your service's page, navigate to **Cross Cluster Replication** and select **Migrate to Cross Cluster**.
+
+   .. note::
+      
+      When you enable CCR on a particular service, you create another service with the same plan, number of nodes, and disk storage.
+
+4. In the **Create a Cross Cluster Replica** view
+   
+   1. Select a cloud provider and a cloud region, define a name for your new service, and select **Continue**.
+   2. Examine the **Service Summary** section and make sure the configuration for your new service meets your expectations. If so, select **Create Cross Cluster Replica**. 
+
+.. topic:: Result
+   
+   CCR has been enabled by connecting your service to another new service, which is now visible in the **Overview** tab under **Cross Cluster replication**.
+
+Enable CCR with CLI
+-------------------
+
+You can enable CCR for your new or existing Aiven for Apache Cassandra service using CLI.
+
+.. note::
+   
+   In this instruction, the :doc:`Aiven CLI client </docs/tools/cli>` is used to interact with Aiven APIs.
+
+.. topic:: Understand parameters to be supplied
+
+   * ``service_to_join_with`` parameter value needs to be set to a name of an existing service in the same project. The supplied service name indicates the service you connect to for enabling CCR. The two connected services create a CCR service pair.
+   * ``cassandra.datacenter`` parameter value needs to be set to a name of a datacenter for your service. Make sure each of the two service constituting a CCR pair belongs to a different datacenter.
```

Comment:
> belong to a different datacenter

this section kind of hints that the datacenter name is supposed to be obtained from the infra or the API, while it's really the user who's supposed to make a choice here... maybe write something like:

`cassandra.datacenter` is the datacenter name used to identify nodes from a particular service in the cluster's topology. In Aiven for Cassandra CCR, nodes of either of the two services belong to a single datacenter, therefore this value needs to be unique for each service. We suggest to set it equal to the service name. You will need this value later on when setting up replication for your keyspaces.